spec/translated_properly_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+  describe('original_text not equal translated_text') do
```

Comment:
Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.